plugins/plugin-annotations/src/title.mjs

```diff
@@ -112,6 +112,13 @@ const title = function (config, { Point, points, scale, locale, store, part, log
    * Title: cutlist
    */
   if (mc.cutlist) {
+    /*
+     * Remove any cut instructions already defined
```

Comment:
This feels a bit hackish 🤔 

It seems to me that the problem is in line 135:

```mjs
const id = `${ids.cutlist}_${material}_${c}`
```

It's creating an arbitrary ID there, rather than using `store.generateMacroIds()`.

A better fix would be to add all of these to the `store.generateMacroIds()` call, so we can keep track of these point IDs.

Reply:
If I'm not being very clear, I can take a stab at this in the weekend.

Reply:
> This feels a bit hackish 🤔
That's what I told you ;-)

Reply:
@joostdecock The `title` macro already stores those `id`s appropriately. You can retrieve them using `store.getMacroIds(mc.id)`. But in an inherited part, you don't know what material was used, and as such, you cannot know if there are any to be removed, how many there are to be removed, and what their name is. 
I can turn the whole shebang into an array in the store, and use that to remove them in any subsequent (higher level) part when called. But I'm not sure that's a more elegant solution. (See in this latest commit)